CONTRIBUTING.md

```diff
@@ -157,9 +157,9 @@ targeting an older kubernetes version. For example, `IngressV1Beta` is not
 available in cdk8s-plus-22, so changing it would require making a change to
 cdk8s-plus-21 and cdk8s-plus-20. If you need to make a pull request to a version
 of cdk8s-plus that isn't the latest version, then **the branch name of your pull
-request must start with `k8s-XX/`** where XX is the version number. For example,
+request must contain `k8s-XX-`** where XX is the version number. For example,
```

Comment:
@Chriscbr FYI this change. I made it so that our dependency upgrade PR's fit the naming convention, otherwise they target by default to `k8s-22`, which messes up the import and so on...WDYT?

Reply:
Ah, yeah that make sense. Why is the hyphen at the end needed though?

Reply:
So I wanted a barrier after the version number, and I couldn't use `/` because it doesn't exist in the dependency upgrade PR's :)